mozci/console/commands/push.py

```diff
@@ -372,7 +358,7 @@ def handle(self) -> None:
                 self.line(
                     "<error>Provided --medium-confidence should be a float.</error>"
                 )
-                return
+                raise
```

Comment:
In these cases where the arguments were wrong we might simply return a non-0 error code instead of raising (the exception will not help with additional info), but this works too.

Reply:
Your pick, I can change it to `exit(1)` if you want

Reply:
:+1: thanks